Helmsman/utils/esignet-standalone-2.0.0/esignet-plugin-values-2-0-0.yaml

```
@@ -0,0 +1,174 @@
+
+#extraEnvVarsSecret: []
+
+#istio:
+#  enabled: true
+#  gateways:
+#    - istio-system/public
+#    - istio-system/internal
+#  prefix: /v1/esignet/
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)